pages/osx/pppd.md

```diff
@@ -0,0 +1,8 @@
+# pppd
+
+> Establish Point-to-Point connection to another computer.
+> It should not be invoked manually.
```

Comment:
I think, this should be moved to common, since it also works on Linux and Solaris.
And it should have a More information too: https://ppp.samba.org/pppd.html